### DISCUSSION TITLE
My MiniDexed incarnation! Diy PCB for Midi, PCM5102A and PS

### DISCUSSION THREAD

**Discussion by CocoFBest** [probonopd/MiniDexed#406] [Show and tell]
Hi guys!

I recently put all the pieces together for my version of MiniDexed.

I used a Raspberry PI 3 A+, a 16x2 generic display with i2c backpack. 

The audio DAC, Power supply and midi in is contained in a DIY pcb I made at my workshop. Also the encoder pcb is self made. 

Aluminum box is a clone of a Hammond 1590DD, knobs are temporary until I can decide for fancier ones. 

![WhatsApp Image 2022-12-20 at 18 19 38](https://user-images.githubusercontent.com/118852891/208777047-2593738e-aea5-4c6b-9d1e-9bb1d96b1a66.jpeg)
![WhatsApp Image 2022-12-20 at 18 19 39](https://user-images.githubusercontent.com/118852891/208777043-79dd91fe-0f1f-443e-9cc2-b1fa8fe20f98.jpeg)
![WhatsApp Image 2022-12-20 at 18 19 39 (2)](https://user-images.githubusercontent.com/118852891/208777038-02e2836f-0b54-409b-9d30-786e3c25eaee.jpeg)
![WhatsApp Image 2022-12-20 at 18 19 40](https://user-images.githubusercontent.com/118852891/208777045-782a6f67-02e8-4a1e-b5d4-73458338adc7.jpeg)
![WhatsApp Image 2022-12-20 at 18 19 39 (3)](https://user-images.githubusercontent.com/118852891/208777041-2903af48-1d80-4726-8222-c6daed257423.jpeg)

Regards

**Comment by diyelectromusic** [comment]
Very nice!  And what a great IO board :)  Was that chemically etched or milled?  Very neat SMD soldering :)

Also, just out of interest, what else is on the PCB - is that an amplifier (and volume pot)?  And although those are stereo jack sockets, it looks like you are using them as two mono (so I assume L/R) outputs?

As I say, very nice :)

Kevin

**Comment by CocoFBest** [comment]
Hi Kevin!!
Thanks a lot.
Pcb is milled with a cheap chinese CNC. I've done a lot chemical etching, but currently I don't have toner transfer paper or silkscreening for the level of detail that this design requires.

I used a poor man SMD soldering technique, mostly with the DAC. You put a blob of solder conecting all the pins, and then you "wipe" off the excess with copper desoldering braid. Works very good. Just in case  I verified after with the multimeter that I didn't had any short circuits to be sure.

On the PCB you have from left to right (top view): dc buck converter to regulate the dc in to 5v. Then you have the DAC on the copper side and all the misc stuff that it needs to work (regulators, etc). Is basically a clone of the Datasheet or the commercial boards. After that goes the Volume Pot to a twin opamp circuit (opamp is NE 5532, underneath) for stereo. That ends in both  6 1/4 jacks. Using the inbuilt switching you could connect the left for mono, or both for stereo. 
And finally you have the 6n138 optocoupler for the MIDI in. 
Thats all, the same as having things separated but condensed for easyness of use 

Regards!
